NuGet.Server.sln

```diff
@@ -22,6 +22,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{0F277539-E9D
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.Server.Core.Tests", "test\NuGet.Server.Core.Tests\NuGet.Server.Core.Tests.csproj", "{FDE8DDFB-E01B-47FD-9945-0A151F3AE589}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.Server.V2", "src\NuGet.Server.V2\NuGet.Server.V2.csproj", "{273E4575-4054-4578-9D14-8D4E640A365F}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.Server.V2.OWinSampleHost", "src\NuGet.Server.V2.OWinSampleHost\NuGet.Server.V2.OWinSampleHost.csproj", "{5CF9DB92-53DE-488E-82F9-4C3F867516B2}"
```

Comment:
Can the project (and folders etc.) be renamed to `NuGet.Server.V2.OwinSampleHost`? "OWin" -> "Owin" basically :-)